doc/release/2.11.0.rst

```diff
@@ -0,0 +1,586 @@
+Tarantool 2.11.0
+================
+
+Released on 2023-05-22.
+
+*   Release: :tarantool-release:`2.11.0`.
+*   Tag: ``2.11.0``.
+
+
+The 2.11 release of Tarantool includes many new features and fixes for Enterprise and Community versions. 2.11 is the long-term support (LTS) release with four years of maintenance. This means that you will continue to receive all the necessary security fixes and bug fixes throughout this period. You can learn more about the Tarantool release policy from the :doc:`corresponding document </release/policy>`.
+
+
```

Comment:
I would extend the introduction a bit: 
+ "find the detailed changelog on GitHub <link>"
+ "Read upgrade instructions in docs <link>"

And also add a ToC to make feature names more visible from the start.

Reply:
I still feel that intro is a bit too brief. The _What's New_ format is new for Tarantool, and so is the combination of CE and EE features in one release document. Thus, to avoid confusion, I would write **explicitly** that:
- this doc provides an overview of important features
- here we describe both CE and EE features
- the complete list of changes and bug fixes now lives only on Github

Also, more words about the upgrade might be useful. A note looks like it's some slightly related info while it's pretty important. I would place the upgrade info either under a subheading or just in a separate paragraph of the intro. Plus, we can think about being more helpful and provide some guidance on upgrade issues, smth like: "file us an issue" or "reach out to support" - PM team should know better.